package.json

```diff
@@ -147,6 +147,8 @@
 		"psl": "^1.8.0",
 		"punycode": "^2.1.1",
 		"query-string": "^7.0.1",
+		"react": "^16.9.0",
+		"react-dom": "^16.9.0",
```

Comment:
Is this necessary?

Reply:
Not really. Tested without it and works just fine. Removed, thank you!